resources/tests/test_resources_queries.py

```diff
@@ -118,6 +119,59 @@ def test_get_berths(berth):
     }
 
 
+def test_get_berth_with_leases(berth, superuser):
+    berth_lease = BerthLeaseFactory()
```

Comment:
you can pass the berth right here: 
```python
berth_lease = BerthLeaseFactory(berth=berth)
```
this way you won't have to assign the attribute on a separate line